integration_tests/e2e/education-support-plan/create/checkYourAnswersChangeLinks.cy.ts

```diff
@@ -111,7 +135,11 @@ context(`Change links on the Check Your Answers page when creating an Education
               "@.prisonId == 'BXI' && " +
               "@.planCreatedBy.name == 'Joe Bloggs' && " +
               "@.planCreatedBy.jobRole == 'Peer Mentor' && " +
-              '@.otherContributors == null && ' +
+              '@.otherContributors.size() == 2 && ' +
```

Comment:
Right at the very end of this cypress test the user clicks Continue to save the Education Support Plan to the API.
The API call is stubbed with wiremock, and we assert the request body sent to assert the correct values were sent to the API